env.example

```diff
@@ -142,6 +142,9 @@ ETHERPAD_SKIN_VARIANTS=super-light-toolbar super-light-editor light-background f
 # Select authentication type: internal, jwt or ldap
 #AUTH_TYPE=internal
 
+# Enable auto login 
+#ENABLE_AUTO_LOGIN=1
```

Comment:
Please don't add it here. Document it on the handbook instead.

Reply:
In the source code, the default value is true, and I thought that adding this comment to _env.example_ was a good practice for others to know what the default value is and how it can be configured without having to read the documentation.

Reply:
But of cause, if this pull request is accepted, I will create a new pull request on the handbook as well, to add this new configuration note.